# Fixes how PostScript::Elements reports its version so we don't trigger a
# "Can't modify constant item in scalar assignment" error when importing.
--- Elements.pm
+++ Elements.pm
@@ -7,11 +7,12 @@
 # itself.
 #
 
-PostScript::Elements::VERSION = '0.06';
-
 package PostScript::Elements;
 use strict;
 
+use vars qw($VERSION);
+$VERSION = '0.06';
+
 # @paramnames is the list of valid parameters,
 # %defaults is the hash of default values for those parameters.
 #
